### PR TITLE
Add Function DefaultAccounting and NewAccounting

### PR DESCRIPTION
**Pull request by Dexus**
and prevent accounting.init() defaults on Accounting Objects from this two functions created

### CODE REVIEW

File: accounting_test.go [hunk 1]
```diff
@@ -19,8 +19,12 @@ func init() {
 	fmt.Printf("version: %s", runtime.Version())
 }
 
+func TestNewAccounting(t *testing.T) {
+
```
Comment (leekchan) [inline accounting_test.go]: Hm... This test contains empty body. Is this PR in progress?

**Comment by leekchan** [comment]
Thanks for your PR. Do you think the new functions are more convenient than the struct-based initialization? I think the struct-based initialization is more intuitive. What do you think?

File: accounting_test.go (hunk 1 above)
Comment (Dexus) [reply accounting_test.go]: Oh, sorry I did add the Checks in TestFormatMoney.

**Comment by Dexus** [comment]
Yes, the new functions are better suited than the structure-based initialization. Both are still possible, but I think it is better to create the objects from the structure with a function. If desired, methods should also be defined to overwrite the "defaults". Because in some countries a space is desired as a thousand separator.

What do you think?

File: accounting_test.go (hunk 1 above)
Comment (leekchan) [reply accounting_test.go]: Okay. Please remove this empty test :)

**Comment by leekchan** [comment]
Okay. It seems to be reasonable 👍 Could you also add some use cases of the new functions in the readme & godoc? (such as using the empty string separator (#8), or something you want)

Thanks :)

**Comment by Dexus** [comment]
All right, I'll do it later, when I have time for it besides my other work.

**Comment by Dexus** [comment]
So now all things are done.

**Comment by leekchan** [comment]
Cool! Thanks :)

**Pull request merged**